contracts/src/factory/HyperdriveFactory.sol

```diff
@@ -730,6 +733,7 @@ contract HyperdriveFactory is IHyperdriveFactory {
     /// @param _deploymentId The deployment ID to use when deploying the pool.
     /// @param _deployerCoordinator The deployer coordinator to use in this
     ///        deployment.
+    /// @param __name The name of the Hyperdrive pool.
```

Comment:
```suggestion
    /// @param __name The name of the Hyperdrive factory.
```

Reply:
for some reason delete button isn't working for this comment, please disregard :)